packages/transactions/src/signer.ts

```diff
@@ -93,7 +102,13 @@ export class TransactionSigner {
     }
 
     const nextSighash = this.transaction.signNextOrigin(this.sigHash, privateKey);
-    this.sigHash = nextSighash;
+
+    if (
+      isSingleSig(this.transaction.auth.spendingCondition) ||
```

Comment:
Probably not necessary

Reply:
Yeah, this is just for typescript.  I can do it with a type cast but I typically try not to. Or keep the raw comparisons which can type infer in TS